src/lib.rs

```diff
@@ -1,8 +1,16 @@
-#[cfg(test)]
```

Comment:
Remove this block could make CI to fail. We can remove them after we have added real code here.

Reply:
> Remove this block could make CI to fail. We can remove them after we have added real code here.

OK, I have updated it, PTAL.